cloud.ganttproject.colloboque/src/main/kotlin/cloud/ganttproject/colloboque/PostgreStorageApi.kt

```diff
@@ -112,6 +117,60 @@ class PostgreStorageApi(private val factory: ConnectionFactory, private val supe
     }
   }
 
+  fun parallelTransactions(
+    projectXml: String,
+    baseTxnId: BaseTxnId,
+    serverTransaction: List<XlogRecord>,
+    clientTransaction: List<XlogRecord>
+  ): Boolean {
+    val settings = Settings().withRenderNameCase(RenderNameCase.LOWER)
+      .withInterpreterDialect(SQLDialect.POSTGRES)
+    val jdbcUrl = "jdbc:postgresql://localhost:5432/${baseTxnId}_${Random.nextLong()};DB_CLOSE_DELAY=-1;DATABASE_TO_LOWER=true"
+    val dataSource = PGSimpleDataSource()
+    dataSource.setURL(jdbcUrl)
+    val database = SqlProjectDatabaseImpl(dataSource, dialect = SQLDialect.POSTGRES)
+    projectFromXml(projectXml, baseTxnId) { database }
+    val clientConnection = dataSource.getConnection("postgres", "")
+    val serverConnection = dataSource.getConnection("postgres", "")
+    val serverDsl = DSL.using(serverConnection, settings)
+    val clientDsl = DSL.using(clientConnection, settings)
+    serverDsl.startTransaction()
```

Comment:
I guess it will start the transaction with READ COMMITTED isolation level. You need to set `isolationLevel` property on the connection instance.